### PR TITLE
Denis/move js2ts

### PR DESCRIPTION
**Pull request by Nebyt**
Changing JS to TS in appium tests.

### CODE REVIEW

**Review by wolverineks — COMMENTED** [review]
what is driver? its not defined anywhere

**Comment by Nebyt** [comment]
> what is driver? its not defined anywhere

The answer from the [webdriver.io](https://webdriver.io/docs/browserobject/)
> The browser object is the session instance you use to control the browser or mobile device with. If you use the WDIO test runner, you can access the WebDriver instance through the global browser or driver object.
> 
> The session is initialized by the test runner. The same goes for ending the session. This is also done by the test runner process.

image

File: tsconfig.json [hunk 1]
```diff
@@ -11,7 +11,16 @@
...
     "noImplicitAny": false,
     "target": "esnext",
-    "types": ["react", "react-native", "react-redux", "jest"],
+    "types": [
+      "react",
+      "react-native",
+      "react-redux",
+      "jest",
+      "node",
+      "webdriverio/sync",
```
Comment (wolverineks) [inline tsconfig.json:20]: the docs mention `webdriverio/async`
is there a difference?

**Comment by wolverineks** [comment]
> > what is driver? its not defined anywhere
> 
> The answer from the [webdriver.io](https://webdriver.io/docs/browserobject/)
> > The browser object is the session instance you use to control the browser or mobile device with. If you use the WDIO test runner, you can access the WebDriver instance through the global browser or driver object.
> > 
> > The session is initialized by the test runner. The same goes for ending the session. This is also done by the test runner process.
> 
> image
> 

I'm seeing a ts error with `driver`, are the types installed correctly?

File: tsconfig.json (hunk 1 above)
Comment (Nebyt) [reply tsconfig.json:20]: Yeah, you are right. `webdriverio/async` should be here.

**Review by wolverineks — APPROVED** [review]

**Review by stackchain — APPROVED** [review]

**Pull request merged**